django-stubs/db/models/expressions.pyi

```diff
@@ -1,7 +1,7 @@
 import datetime
 from collections.abc import Callable, Iterable, Iterator, Mapping, Sequence
 from decimal import Decimal
-from typing import Any, ClassVar, Generic, Literal, TypeVar
+from typing import Any, ClassVar, Generic, Literal, NoReturn, TypeVar
```

Comment:
Please, use `Never`

Reply:
Is there a specific reason to do so? I know they are equivalent but I like sticking to `NoReturn` for return values. What do you think?

Reply:
OK :)